docs/docs/integrations/libraries/dbt/dbt-cloud.md

```diff
@@ -15,15 +15,24 @@ import Beta from '@site/docs/partials/\_Beta.md';
 
 <Beta />
 
-<p>{frontMatter.description}</p>
```

Comment:
I would keep description here, since it tells the user what this integration does